website/docs/cloud-docs/agents/agents.mdx

```diff
@@ -0,0 +1,102 @@
+1. Verify the integrity of the downloaded archive, as well as the signature of the `SHA256SUMS` file using the instructions available on [HashiCorp's security page](https://www.hashicorp.com/security).
+1. Extract the release archive. The `unzip` utility is available on most Linux distributions and may be invoked as `unzip <archive file>`. Two individual binaries will be extracted (`tfc-agent` and `tfc-agent-core`). These binaries _must_ reside in the same directory for the agent to function properly.
+
+### Updates
+
+By default, the agent automatically updates itself to the latest minor version. Administrators must update the host operating system and all other installed software.
+
+To customize this update behavior, pass the flag `-auto-update` or set the environment variable `TFC_AGENT_AUTO_UPDATE` to one of the following settings.
+
+| Update Setting | Behavior                                                                                                     |
+| -------------- | ------------------------------------------------------------------------------------------------------------ |
+| `minor`        | Matches the default behavior, automatically update the agent to the latest minor version.                    |
+| `patch`        | The agent will only be updated to the newest patch version, new minor versions will require a manual update. |
+| `disabled`     | Disables automatic updates, all updates will be manual.
+
+## Start the Agent
+
+To start the agent and connect it to a Terraform Cloud agent pool:
+
+1. Retrieve the [token](/cloud-docs/agents/agent-pools#create-an-agent-pool) from the Terraform Cloud agent pool you want to use.
+1. Set the `TFC_AGENT_TOKEN` environment variable.
+1. (Optional) Set the `TFC_AGENT_NAME` environment variable. This name is for your reference only. The agent ID appears in logs and API requests.
+
+```
+export TFC_AGENT_TOKEN=your-token
+export TFC_AGENT_NAME=your-agent-name
+./tfc-agent
+```
+Once complete, your agent and its status appear on the **Agents** page in the Terraform Cloud UI. Workspaces can now use this agent pool for runs. Refer to [Configure Workspaces to Use the Agent](/cloud-docs/agents/agent-pools#configure-workspaces-to-use-the-agent) for details.
+
+### Optional Configuration: Run an Agent Using Docker
+
+Alternatively, you can use our official agent Docker container to run the agent.
+
+```
+docker pull hashicorp/tfc-agent:latest
+docker run -e TFC_AGENT_TOKEN=your-token -e
+TFC_AGENT_NAME=your-agent-name hashicorp/tfc-agent
+```
+
+This Docker image executes the tfc-agent process as the non-root tfc-agent user. For some workflows, such as those that require the ability to install software via apt-get during local-exec scripts, you may need to build a customized version of the agent Docker image for your internal use.
```

Comment:
```suggestion
This Docker image executes the `tfc-agent` process as the non-root `tfc-agent` user. For some workflows, such as workflows requiring the ability to install software using `apt-get` during `local-exec` scripts, you may need to build a customized version of the agent Docker image for your internal use.
```
Suggest putting instances of tfc-agent, apt-get, and local-exec in backticks to set them apart.

Suggest changing "via" to "using" to avoid Latin words.